client/app/web/modules/series/root/MasterSeries.js

```diff
@@ -1,13 +1,14 @@
-import React from 'react'
+import React from 'react';
 import SeriesGrid from './masterSeries/SeriesGrid';
 import SeriesSearchSF from '../../../../modules/series/root/masterSeries/SeriesSearchSF';
 import SeriesSearch from './masterSeries/SeriesSearch';
+import masterStyles from './masterSeries.scss';
 
 const MasterSeries = props => 
-    <div>
+    <main className={masterStyles.masterFlex}>
```

Comment:
Be consistent in code style. I think i would add a new rule to avoid all this annoying comments